ext/memcached_rb/memcached_rb.c

```diff
@@ -1,460 +1,518 @@
-#include <memcached_rb.h>
-#include <memcached_rb_behavior.h>
+#include "memcached_rb.h"
 
-VALUE rb_mMemcached,
-  rb_cServer,
-  rb_eMemcachedError;
+VALUE rb_mMemcached;
 
-ID id_ivar_hostname,
```

Comment:
Most of these IDs don't need to be cached aggressively. The ones for the `Server` class are gone because the `Server` class is gone! (check below)